AutomationUtils/Tests/FeatureFilesTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using AutomationUtils.Utils;
+using NUnit.Framework;
+
+namespace AutomationUtils.Tests
+{
+    [TestFixture]
+    public class FeatureFilesTests
+    {
+        [Test]
+        public void Check_AllFeatureFilesAndTheirContent_CountOfFiles()
+        {
+            var featureFilesContent = TestsUtils.AllFeatureFilesAndTheirContent();
+            Verify.AreEqual(2, featureFilesContent.Count,
+                "AllFeatureFilesAndTheirContent method returns incorrect count of files");
+        }
+
+        [Test]
+        public void Check_AllFeatureFilesAndTheirContent_FileNames()
+        {
+            var featureFilesContent = TestsUtils.AllFeatureFilesAndTheirContent();
+            Verify.AreEqual("TestFeatureFile1.feature", Path.GetFileName(featureFilesContent.ElementAt(0).Key),
```

Comment:
If somebody will add more FF this order can also cahnged and your tests will fail